tools/devctr/Dockerfile

```diff
@@ -147,6 +149,16 @@ RUN mkdir "$TMP_BUILD_DIR" && cd "$TMP_BUILD_DIR" \
     && cd / \
     && rm -rf "$TMP_BUILD_DIR"
 
+# Build iperf3. We need it until 3.18 is released with a fix to a crash on aarch64.
```

Comment:
I thought about that as well and decided against it because it's mainly us who maintain the devctr rather than the community so we can track that internally. The only reason for a GH issue could be that we put a more detailed explanation of the problem. Happy to create it if you think it's worth it.
In any case, I'm running testing on top of https://github.com/pb8o/firecracker/tree/ci-no-ssh-keys to see if the issue goes somehow without changing iperf3 in devctr.

Reply:
The test failed in https://github.com/firecracker-microvm/firecracker/pull/4858 , so un-wipping this PR.

Reply:
no strong opinion on that. ship it!